### ISSUE TITLE
Private repos crash Yanpm

### ISSUE THREAD

**Issue by valgaze** [jstty/yanpm#2]
When using a private package, yanpm installs the package correctly but will hang and exit out. 

The issue appears to be that the "package" key is referencing the URL to the package instead of the package name (and the associated directory in node_modules), see line 676 here: https://github.com/jstty/yanpm/blob/master/lib/manager.js#L676

One idea for a fix might be to have a private repo flag to handle this case.

**Comment by valgaze** [comment]
Failing example:

```javascript
var yanpm = require('yanpm');
var ya = new yanpm();
var config = [{
  "name": "someName",
  "package": "git+ssh://git@enterpriseco.com:Organization/somerepo.git#branch/v1.0"
}];
ya.add(config)
  .load(function() {
    try {
      var packageRef = ya.get('someName'); //This nulls out
      packageRef.someMethod();
    } catch (e) {
      console.log('**ERROR:', e);
    }
  });
```

**Comment by jstty** [comment]
This issue should be resolved in c308dd8809d58b28a1cb732a4701f7350a9a8685 (v0.7.0)
